source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,129 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later.
+
+   You should only use the Versioned API feature if all the MongoDB
+   servers you are connecting to support this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with behavior compatible with the
+specified **API version**. An API version defines the expected behavior of the
+operations it covers and the format of server responses. If you change to
+a different API version, the operations are not guaranteed to be
+compatible and the server responses are not guaranteed to be similar.
+
+When you use the Versioned API feature with an official MongoDB driver, you
+can update your driver or server without worrying about backward compatibility
+issues of the commands covered by the Versioned API.
+
+See the server manual page on `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__
+for more information including a list of commands it covers.
+
+The following sections describe how you can enable Versioned API for
+your MongoDB client and the options that you can specify.
+
+Enable Versioned API on a MongoDB Client
+----------------------------------------
+
+To enable the Versioned API, you must specify an API version in the settings
+that you use to create your MongoDB client. Once you instantiate a
+``MongoClient`` instance with a specified Versioned API, all commands you
+run with that client use that version of the Versioned API. If you need to
+run commands not covered by the Versioned API or with a different version of
+it, instantiate a separate client.
+
+The example below shows how you can instantiate a ``MongoClient`` that
+sets the Versioned API version and connects to a server by performing the
+following operations:
+
+- Construct a ``ServerApi`` instance using the ``ServerApi.Builder``
+  helper class.
+- Specify a Versioned API version using a constant from the
+  ``ServerApiVersion`` class.
+- Construct a ``MongoClientSettings`` instance using the
+  ``MongoClientSettings.Builder`` class.
+- Specify a server to connect to using a ``ServerAddress`` instance.
+- Instantiate a ``MongoClient`` using the ``MongoClients.create()`` method
+  and pass your ``MongoClientSettings`` instance as a parameter.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/VersionedApiExample.java
+   :start-after: start serverAPIVersion
+   :end-before: end serverAPIVersion
+   :language: java
+   :dedent:
+
+.. warning::
+
+   If you specify an API version and connect to a MongoDB server that does
+   not support Versioned API, your application may raise an exception when
```

Comment:
Nit: Saying "may raise" makes me wonder why it might or might not be raised.
```suggestion
   not support Versioned API, your application raises an exception when
```

Reply:
This is based on my observation rather than any rule in the Versioned API spec, so I think it would be better not to make it a definitive statement.